src/use_case_classification.Rmd

```diff
@@ -0,0 +1,147 @@
+# Use Cases
+
+## Classification
+
+One popular example regarding a classification task is the "Titanic" showcase. We have different passenger information - like name, age or fare - available with the aim to predict which kind of people would have survived the titanic sinking.
+
+Therefore we load the titanic dataset and other libraries that are needed for this use case.
+
+```{r, results='hide', message=FALSE, warning=FALSE}
+library(titanic)
+library(mlr)
+library(BBmisc)
+```
+
+
+```{r}
+data = titanic_train
+head(data)
+```
```

Comment:
I'm using this data set for training and testing and thought that "data" is not so confusing than titanic_train.